tests/service/data/SyntaxTree/OpenDeclaration/E_openInTypeDecl.fs

```diff
@@ -0,0 +1,63 @@
+// #Regression #Conformance #DeclarationElements #Import 
```

Comment:
Inspecting the AST is very hard in larger files. Could you please split this into multiple files(covering each scenario individually) ?

Reply:
ok